helper-cli/src/main/kotlin/common/Utils.kt

```diff
@@ -254,7 +254,10 @@ internal fun OrtResult.processAllCopyrightStatements(
         licenseInfoResolver.resolveLicenseInfo(id).forEach innerForEach@{ resolvedLicense ->
             if (omitExcluded && resolvedLicense.isDetectedExcluded) return@innerForEach
 
-            val copyrights = resolvedLicense.getResolvedCopyrights(omitExcluded).flatMap { resolvedCopyright ->
+            val copyrights = resolvedLicense.getResolvedCopyrights(
```

Comment:
Commit message:

> Remove the BTW wrong processing of copyrights

Does "BTW" stand for "by the way"? If so, the title wording sounds a bit odd to me. Can you think of a way to reword it without using an abbreviation?

> This fixes and issue

"an issue"

> The reports can be computed in about 10 minutes.

Maybe "can *now* be computed"?

Generally, I'm wondering why the (seemingly rather sophisticated) tests that we have in this area apparently did not catch the issues described in the commit message. Could you maybe also add a comment on that to the commit message?

Reply:
All done.